src/phylum/ci/ci_base.py

```diff
@@ -349,15 +349,16 @@ def ensure_project_exists(self) -> None:
         if self.phylum_group:
             print(f" [-] Using Phylum group: {self.phylum_group}")
             cmd = [str(self.cli_path), "project", "create", "--group", self.phylum_group, self.phylum_project]
-        ret = subprocess.run(cmd, check=False, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
+        ret = subprocess.run(cmd, check=False, stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)  # noqa: S603
         # The Phylum CLI will return a unique error code of 14 when a project that already
         # exists is attempted to be created. This situation is recognized and allowed to happen
         # since it means the project exists as expected. Any other exit code is an error.
+        cli_exit_code_project_already_exists = 14
```

Comment:
This is perfectly fine if this is the only instance of an exit code error from CLI that we check, but I'd refactor this in a global enum if we were to do this in a few places. (EDIT: I see we have this below as well; I'd leave it as it is for the time being, and address this in a separate PR, though not with super high priority).

Reply:
Roger that...agreed. There was a thought about doing something like that (an `IntEnum` for CLI exit codes) but the same conclusion was reached...it isn't used enough to justify it (at least IMO).